sdk/src/test/scala/com/horizen/account/api/http/AccountEthRpcRouteMock.scala

```diff
@@ -114,6 +116,19 @@ abstract class AccountEthRpcRouteMock extends AnyWordSpec with Matchers with Sca
   })
   val mockedSidechainTransactionActorRef: ActorRef = mockedSidechainTransactionActor.ref
 
+  val mockedSyncStatusActor = TestProbe()
+  mockedSyncStatusActor.setAutoPilot(new testkit.TestActor.AutoPilot {
+    override def run(sender: ActorRef, msg: Any): TestActor.AutoPilot = {
+      msg match {
+        case ReturnSyncStatus() =>
+          if (sidechainApiMockConfiguration.getShould_syncStatusActor_ReturnSyncStatus_reply()) sender ! new SyncStatus(false)
```

Comment:
check also `setShould_syncStatusActor_ReturnSyncStatus_reply`
so far it is declared but not used.
Use this mock in all the places where it is needed.
see comment below.

Reply:
As reported in the comment below I did a refactor of the ActorRef declaration without mocking any logic because here we test only the rpc interaction at higher level, the `setShould_syncStatusActor_ReturnSyncStatus_reply` flag is no longer used so it was removed.